### ISSUE TITLE
Docker install broken

### ISSUE THREAD

**Issue by kellrott** [galaxyproject/ansible-galaxy-os#3]
Using under the planemo-machine build, docker fails with the error:

```
    googlecompute: TASK: [galaxyprojectdotorg.galaxy-os | Add Docker repository key] *************
    googlecompute: failed: [localhost] => {"failed": true}
    googlecompute: msg: Failed to download key at https://get.docker.io/gpg: Request failed: <urlopen error [Errno 1] _ssl.c:510: error:14094410:SSL routines:SSL3_READ_BYTES:sslv3 alert handshake failure>
    googlecompute: 
    googlecompute: FATAL: all hosts have already failed -- aborting
    googlecompute:
    googlecompute: PLAY RECAP ********************************************************************
```

**Comment by afgane** [comment]
I got a fix for this locally - let me clean it up and commit it in a minute.

**Comment by afgane** [comment]
Should be fixed in 58c3b61ab37672146a46d95f0671337f39b13a31
